File: Arcade/Intro/19.cpp
bool areEquallyStrong(int yourLeft, int yourRight, int friendsLeft, int friendsRight) {
  // areEquallyStrong : Int, Int, Int, Int -> Bool
  // Each input represent the strength of an arm, see if they are
  // equivalent strong.
  return ((yourLeft == friendsLeft) && (yourRight == friendsRight)) ||
    ((yourRight == friendsLeft) && (yourLeft == friendsRight));
}

File: Arcade/Intro/19.clj
(defn areEquallyStrong [yourLeft yourRight friendsLeft friendsRight]
  "areEquallyStrong : Int, Int, Int, Int -> Bool
   Each input represent the strength of an arm, see if they are
   equivalent strong."
  (or (and (= yourLeft friendsLeft) (= yourRight friendsRight))
      (and (= yourLeft friendsRight) (= yourRight friendsLeft))))
